psi4/src/psi4/liboptions/liboptions.cc

```diff
@@ -837,7 +837,7 @@ Data& Options::use(std::string& key) {
         }
 
         std::string choices_joined;
-        std::accumulate(std::begin(choices), std::end(choices), 0, [&choices_joined](int&, std::string& s) {
+        std::accumulate(std::begin(choices), std::end(choices), 0, [&choices_joined](int, std::string& s) {
```

Comment:
Why not just do this, which expresses the intention more clearly? There is no need to use accumulate, since we're not accumulating anything here.
```suggestion
for (const std::string& s : choices) {
    if (!choices_joined.empty()) {
        choices_joined.append(" ");
    }
    choices_joined.append(s);
}
```